services/horizon/docker/docker-compose.yml

```
@@ -50,6 +52,8 @@ services:
     environment:
       - DATABASE_URL=postgres://postgres@host.docker.internal:5432/horizon?sslmode=disable
       - STELLAR_CORE_DATABASE_URL=postgres://postgres:mysecretpassword@host.docker.internal:5641/stellar?sslmode=disable
+      # to connect to the public stellar network
+      # use "Public Global Stellar Network ; September 2015" of "Test SDF Network ; September 2015"
```

We could leave the line there ready to be uncommented for them.
```suggestion
      #- NETWORK_PASSPHRASE=Public Global Stellar Network ; September 2015
```